runner/execer/os/easy_test.go

```diff
@@ -76,9 +76,9 @@ func TestOutput(t *testing.T) {
 }
 
 func TestMemUsage(t *testing.T) {
-	// Command to increase memory by 10MB every .1s until we hit 100MB after 1s.
+	// Command to increase memory by 1MB every .1s until we hit 50MB after 5s.
```

Comment:
What's the motivation behind these changes?

Reply:
These tests fail when my macbook has a high load.